synapse/storage/events_worker.py

```diff
@@ -12,6 +12,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from __future__ import division
```

Comment:
Is this necessary given we seem to only be doing `//` in here?

Reply:
maybe not, I don't think